modules/MIPS/SQL/PQRS_direct_entry_table.sql

```diff
@@ -1,3 +1,17 @@
+/* Copyright (C) 2015 - 2017      Suncoast Connection
+ * 
+ * LICENSE: This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0
+ * See the Mozilla Public License for more details. 
+ * If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ * 
+ * @author  Art Eaton <art@suncoastconnection.com>
+ * @author  Bryan lee <leebc@suncoastconnection.com>
+ * @package LibreHealthEHR 
+ * @link    http://suncoastconnection.com
+ * @link    http://librehealth.io
+ *
+ * Please support this product by sharing your changes with the LibreHealth.io community.
+ */
 DROP TABLE IF EXISTS pqrs_direct_entry_lookup;
 CREATE TABLE IF NOT EXISTS `pqrs_direct_entry_lookup` (
```

Comment:
Just curious if we drop the table then do we need to check to make sure that it does not exist?